src/v3/Tracer.ts

```diff
@@ -200,41 +208,62 @@ export class Tracer<
       RelationSchemasT,
       VariantsT
     >,
+    opts?: {
+      noDraftPresentBehavior: 'error' | 'warning' | 'noop'
+    },
```

Comment:
since we're using this type in multiple places, can we extract it into a shared interface?